client/Cargo.toml

```diff
@@ -28,5 +28,6 @@ thiserror = { workspace = true }
 tokio = { workspace = true, features = ["fs", "rt-multi-thread"] }
 toml = "0.7.6"
 tonic = { workspace = true, features = ["tls"] }
-tower = "0.4.13"
+tower = { version = "0.5.2", features = ["util"] }
 x509-certificate = "0.18.0"
+hyper-util = "0.1.6"
```

Comment:
Is there a reason not to use `hyper-util = "0.1.10"`?

Reply:
Nope! Likely a `cargo add` with the old rust version or something like that.  Updated in https://github.com/aurae-runtime/aurae/pull/541/commits/9b07800c2e6f85053bb45d813d5c422a32a44685